sample-config/controller.toml

```diff
@@ -0,0 +1,7 @@
+[nats]
+hosts = ["nats"]
+
+# An optional token is allowed for token authentication.
+auth = { token = "my-secret-token" }
```

Comment:
rename to "nats-secret-token" here and in docker-compose? to clarify that they have to be the same?

Reply:
or even a comment line saying "note this must be set on server", (its obvious I know, but I speak from experience setting up synapse and stuff while knowing nothing, we all start somewhere!)

Reply:
hmm fair, I might just comment this out and not set a token on the demo server so that when we have a CLI it can default to not using token auth